### PR TITLE
Fixed wing TECS controller preparation (stage 1)

### PR DESCRIPTION
**Pull request by digitalentity**
Preparing to bring TECS to INAV - controll altitude and airspeed with pitch and throttle.
Paper on TECS: [tecs_autopilot.pdf](https://github.com/iNavFlight/inav/files/1202998/tecs_autopilot.pdf)

Change FW altitude controller to operate energy balance instead of raw altitude. PIDs or behaviour shoudln't change.

### CODE REVIEW

File: src/main/navigation/navigation.c [hunk 1]
```diff
@@ -2605,9 +2605,9 @@ void navigationUsePIDs(void)
                                         (float)pidProfile()->bank_fw.pid[PID_POS_XY].I / 100.0f,
                                         (float)pidProfile()->bank_fw.pid[PID_POS_XY].D / 100.0f);
 
-    navPidInit(&posControl.pids.fw_alt, (float)pidProfile()->bank_fw.pid[PID_POS_Z].P / 100.0f,
-                                        (float)pidProfile()->bank_fw.pid[PID_POS_Z].I / 100.0f,
-                                        (float)pidProfile()->bank_fw.pid[PID_POS_Z].D / 100.0f);
+    navPidInit(&posControl.pids.fw_alt, (float)pidProfile()->bank_fw.pid[PID_POS_Z].P / 9.80665f,
```
Comment (fiam) [inline src/main/navigation/navigation.c:2614]: Why not GRAVITY_MSS rather than a magic number?

Comment (digitalentity) [reply src/main/navigation/navigation.c:2614]: It's equals to gravity by co-incidence due to math changes to PID itself. I don't want confusion - here it's indeed a magic dimensionless number,

Comment (fiam) [reply src/main/navigation/navigation.c:2614]: Thanks for the explanation. Now that I took the time to better understand the PID handling code, it makes sense.

**Comment by digitalentity** [comment]
Tested, functions the same way as the old code

**Pull request merged**